zenduty/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
Since there are no metrics for this integration, the metadata file can be removed